.docker/populate-services.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+set -e
+
+HTTP_PORT_GATEWAY="${HTTP_PORT_GATEWAY:-8081}"
+
+# Populate the content services
+
+content_services=(
+    blog-articles
+    scholarly-articles
+)
+for service in "${content_services[@]}"; do
+    echo "Populating ${service}"
+
+    files=()
+    while IFS='' read -r line; do files+=("$line"); done < <(find ".docker/${service}/data" -name "*.xml" -print | sort)
```

Comment:
need to run `shellcheck .docker/*.sh` during the build, iterations and complex processes in bash get tricky